web/access_control.go

```diff
@@ -2,15 +2,14 @@ package web
 
 import (
 	"context"
+	"errors"
 
-	"google.golang.org/grpc/codes"
-	"google.golang.org/grpc/status"
-
+	"github.com/bufbuild/connect-go"
 	"github.com/quickfeed/quickfeed/qf"
 	"github.com/quickfeed/quickfeed/web/auth"
 )
 
-var ErrMissingInstallation = status.Error(codes.PermissionDenied, "github application is not installed on the course organization")
+var ErrMissingInstallation = connect.NewError(connect.CodePermissionDenied, errors.New("github application is not installed on the course organization"))
```

Comment:
This is use several places where we need to use the `scm`. Suggest that instead the relevant `getSCM*` functions return this error directly so that we can just `return nil, err` instead of:
```go
	return nil, ErrMissingInstallation
```
There is also one case in `rebuild.go` where we don't use this error, but probably should. But would be solved by returning the error from the source directly.